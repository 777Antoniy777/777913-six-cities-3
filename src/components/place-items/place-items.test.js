import React from 'react';
import renderer from "react-test-renderer";
import PlaceItems from './place-items';

// set mocha data
const items = [
  `item-1`,
  `item-2`,
];

it(`render PlaceItems`, () => {
  const tree = renderer.create(
      <PlaceItems
        items={items}
      />)
      .toJSON();

  expect(tree).toMatchSnapshot();
});
