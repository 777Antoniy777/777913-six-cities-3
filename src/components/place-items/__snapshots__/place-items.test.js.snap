// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render PlaceItems 1`] = `
<ul
  className="property__inside-list"
>
  <li
    className="property__inside-item"
  >
    item-1
  </li>
  <li
    className="property__inside-item"
  >
    item-2
  </li>
</ul>
`;
